src/test/groovy/com/gradle/TestDevelocityInjection.groovy

```diff
@@ -109,6 +109,32 @@ class TestDevelocityInjection extends BaseInitScriptTest {
             .findAll {testGradle, testDvPlugin -> testDvPlugin.compatibleCCUDVersion != null}
     }
 
+    def "applies CCUD plugin via init script with DV plugin already applied using develocity.buildCache"() {
+        given:
+        declareDvPluginApplication(gradleVersion, testDvPlugin)
+        settingsFile << """
+buildCache {
+    remote(${buildCacheExtension}.buildCache)
+}
+"""
```

Comment:
Will Groovy not trim the indent? Even so, Gradle won't care that the indentation is there.

Reply:
👍🏼 changed